packages/@tinacms/fields/src/NumberInput.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+
+Copyright 2019 Forestry.io Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+import * as React from 'react'
+
+interface NumberFieldProps {
+  label: string
+  name: string
+  component: string
+  steps: number | string
+}
+
+export interface NumberProps {
+  name: string
+  input: any
+  field: NumberFieldProps
+  disabled?: boolean
```

Comment:
This seems unused
```suggestion
```

Reply:
Alright ... I made this a quick PR as I haven't gone through the code. Was some time since I worked on this. =)